bittensor/_cli/cli_impl.py

```diff
@@ -484,9 +515,34 @@ def overview(self):
         table.caption = "[white]Wallet balance: [green]\u03C4" + str(balance.tao)
 
         console.clear()
+
+        sort_by: str = self.config.wallet.sort_by
+        sort_order: str = self.config.wallet.sort_order
+
+        if sort_by != "":
+            column_to_sort_by: int = 0
+            sort_descending: bool = False # Default sort_order to ascending
+
+            for index, column in zip(range(len(table.columns)), table.columns):
+                # Fuzzy match the column name. Default to the first column.
+                if fuzz.ratio(sort_by.lower(), column.header.lower().replace('[overline white]', '')) > 80:
+                    column_to_sort_by = index
+                    break
+            
+            if sort_order.lower() == 'desc' or sort_order.lower() == 'descending' or sort_order.lower() == 'reverse':
```

Comment:
nit: would be easier to read if you're checking set membership i.e. `if sort_order.lower() in ('desc', .....)`

Reply:
done